community/modules/scheduler/schedmd-slurm-gcp-v6-controller/variables.tf

```diff
@@ -612,3 +612,31 @@ EOD
   type        = any
   default     = false
 }
+
+variable "universe_domain" {
+  description = "Domain address for alternate API universe"
+  type        = string
+  default     = "googleapis.com"
+  nullable    = false
+}
+
+variable "endpoint_versions" {
+  description = "Version of the API to use (The compute service is the only API currently supported)"
+  type = object({
+    compute = string
+  })
+  default = {
+    compute = null
+  }
+}
+
+variable "gcloud_executable" {
+  description = "Location of the gcloud executable to be used during cleanup"
+  type        = string
+  default     = "gcloud"
+
+  validation {
```

Comment:
I wouldn't make an assumption that the machine that does deployment is the same that would do destroy.
Also `enable_cleanup` can be disable, so this check is excessive. Let's remove it.

Reply:
I agree with this, and I think my latest changes address this so that most users will never need to use the added `gcloud_override_path`, but this new change just adds to the path, so it should still search correctly.